packages/suite-web/e2e/tests/settings/coins.test.ts

```diff
@@ -39,6 +39,7 @@ describe('Coin Settings', () => {
             'tgor',
             'txrp',
             'tada',
+            'dsol',
```

Comment:
This is now failing because sol/dsol are not merged in FW so not available for e2e tests. Let's wait or comment it out till FW is merged 🙏  Then `sol` should be here in the list too.

Reply:
TODO